### ISSUE TITLE
typing stubs and py.typed not included with version 0.20.2

### ISSUE THREAD

**Issue by Akuli** [tree-sitter/py-tree-sitter#153]
This library should now come with stub files, but it doesn't. If I install the latest tree_sitter (0.20.2), mypy doesn't find any stub files:

```
$ pip install mypy tree_sitter
Collecting mypy
  Using cached mypy-1.5.1-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (12.0 MB)
Collecting tree_sitter
  Downloading tree_sitter-0.20.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (486 kB)
     ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 486.4/486.4 kB 635.7 kB/s eta 0:00:00
Collecting typing-extensions>=4.1.0
  Using cached typing_extensions-4.7.1-py3-none-any.whl (33 kB)
Collecting mypy-extensions>=1.0.0
  Using cached mypy_extensions-1.0.0-py3-none-any.whl (4.7 kB)
Installing collected packages: typing-extensions, tree_sitter, mypy-extensions, mypy
Successfully installed mypy-1.5.1 mypy-extensions-1.0.0 tree_sitter-0.20.2 typing-extensions-4.7.1

$ mypy -c "import tree_sitter"
<string>:1: error: Library stubs not installed for "tree_sitter"  [import]
<string>:1: note: Hint: "python3 -m pip install types-tree-sitter"
<string>:1: note: (or run "mypy --install-types" to install all missing stub packages)
<string>:1: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports
Found 1 error in 1 file (checked 1 source file)
```

This was already fixed in a0ceeb1e50b1ecf5b26924b567da99eb58a4b401 but the fix hasn't been released yet. Are there plans to make a new release soon?

**Comment by amaanq** [comment]
Yes, once we release a new ts version (soon I hope...god downstream needs it)

**Comment by amaanq** [comment]
The core release is being blocked by stuff out of my control, I released 0.20.3 but fumbled the CI action so wheels weren't available on PyPI - 0.20.4 is now fixed
